packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -321,6 +321,16 @@ class FileCollection extends DocumentCollection {
     }
   }
 
+  /***
+   * Update the io.cozy.files
+   * Used by StackLink to support CozyClient.save({file})
+   * @param {*} file io.cozy.files
```

Comment:
You can use the typedef defined above for FileAttributes here.

https://github.com/cozy/cozy-client/blob/feat/ExposeFileUpdate/packages/cozy-stack-client/src/FileCollection.js#L21

What does `*` means here ?